test.go

```diff
@@ -0,0 +1,13 @@
+package main
```

Comment:
This is not a test in golang. Take a look at this one https://blog.golang.org/examples